### ISSUE TITLE
Add Links | Infinit loading

### ISSUE THREAD

**Issue by ItJustFox** [JulianPrieber/llc-docker#46]
Platform: `Unraid Version: 6.11.5 `
Server type: `Docker Image - julianprieber/littlelink-custom:latest | Apache/2.4.55 (Unix)`
PHP version: `8.1`

[//]: # (Please make sure you have read the system requirements at https://llc.ovh/req before submitting your issue.)

On **Opera GX**, add Links doesn't work and just keep on loading infinitly, while when I go on regular Chrome It work without any problem.
![image](https://user-images.githubusercontent.com/42358087/218858754-a34ee539-cd9f-4b72-b4ff-b41e2b4b314c.png)

**Comment by JulianPrieber** [comment]
This is most likely caused by an invalid reverse proxy setup.

See:
https://github.com/JulianPrieber/llc-docker/issues/38
https://github.com/JulianPrieber/llc-docker/issues/28

**Comment by ItJustFox** [comment]
> This is most likely caused by an invalid reverse proxy setup.
> 
> See: #38 #28

Actually no ^^' 
It work well fine everywhere, it's just that OperaGX same to don't work only when trying to add new links, like said If I go on chrome or on my phone it work very well

**Comment by ItJustFox** [comment]
![image](https://user-images.githubusercontent.com/42358087/218863951-6b6057d7-23d6-4570-b5de-031e16cc0974.png)

Don't know if that may help a lot ^^'

**Comment by JulianPrieber** [comment]
Mmmmm. I'm not sure what's going on there. We could work with this issue, mainly if it was the same on other browsers as well.
If it works on Chrome, I see no reason why it shouldn't work on Opera GX.

I'm just guessing at this point, but Opera GX has a built-in Ad blocker, right?
Maybe disable that and see if there is any script blocker.

**Comment by ItJustFox** [comment]
> Mmmmm. I'm not sure what's going on there. We could work with this issue, mainly if it was the same on other browsers as well. If it works on Chrome, I see no reason why it shouldn't work on Opera GX.
> 
> I'm just guessing at this point, but Opera GX has a built-in Ad blocker, right? Maybe disable that and see if there is any script blocker.

Yes, it has vpn adblocker selfbuild on it, but their are all aleadry disabled on mine :/ 
I see no reason why it does not work :x
At least you can still try on your side if one day you install Opera GX, and tbh it's not a big problem, like, I just temporarily switch to Chrome to all my links then go back to Opera GX

**Comment by JulianPrieber** [comment]
Wait, could you try creating an account on the [demo instance](https://demo.littlelink-custom.com/) using opera gx and see if it works there?

**Comment by ItJustFox** [comment]
> Wait, could you try creating an account on the [demo instance](https://demo.littlelink-custom.com/) using opera gx and see if it works there?

Hey there, just tested it on Demo instance and it same to work nice without problem ?

**Comment by JulianPrieber** [comment]
This might actually be an invalid reverse proxy

**Comment by ItJustFox** [comment]
> This might actually be an invalid reverse proxy

Then if it's an reverse proxy invalid, why the hell does it work fine on Chrome but not Opera ? :x
I do use Nginx Proxy Manager, and setuped it to use the https://[IP]:4881/ (Unraid 443 -> 4881)

**Comment by JulianPrieber** [comment]
You could try it with Opera GX on https://[IP]:4881 to make sure.
